pkgs/development/python-modules/scipy/default.nix

```diff
@@ -82,7 +90,17 @@ buildPythonPackage rec {
     blas = numpy.blas;
   };
 
-  setupPyBuildFlags = [ "--fcompiler='gnu95'" ];
```

Comment:
@FRidh Git says you authored this. AFAIU this was a no-op since `format = "pyproject"`, and it would have been a no-op now with `format = "build"`. I do not know what this option was originally introduced for and whether I should restore it

Reply:
We're not using setuptools anymore so this shouldn't do a thing.